content/articles/coming-soon-landing-page-using-javascript-timers/index.md

```diff
@@ -0,0 +1,463 @@
+                class="card-img-top"
+              />
+              <div class="card-body">
+                <h5 class="card-title">ALBUM 02</h5>
+                <p class="card-text">Lorem ipsum dolor sit amet consectetur</p>
+                <a href="#" class="btn btn-primary d-block">Listen Now</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="container">
+        <div class="row">
+          <div class="col-md-12">
+            <p class="lead text-center">Support My Music</p>
+            <h4 class="display-4 text-center" style="letter-spacing: 12px;">
+              COMING SOON
+            </h4>
+            <hr />
+            <p id="launch" class="lead"></p>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <script src="js/main.js"></script>
+  </body>
+</html>
+```
+
+**CSS SECTION**
+- style.css
+```css
+header {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url(../img/background.jpg);
+  height: 100vh;
+  background-position: center;
+  background-size: cover;
+  color: #fff;
+}
+.card img {
+  border-radius: 30% 70% 0% 100% / 0% 0% 100% 100%;
+}
+hr {
+  width: 50%;
+  margin: 30px auto;
+  border: #f0f0ea solid 1.5px;
+}
+#launch {
+  text-align: center;
+  font-size: 40px;
+}
+```
+
+**JAVSCRIPT SECTION**
+- main.js
+```javascript
+let countDate = new Date("sep 1,2021 00:00:00").getTime();
+
+let x = setInterval(() => {
+  let now = new Date().getTime();
+
+  let dist = countDate - now;
+
+  let days = Math.floor(dist / (1000 * 60 * 60 * 24));
+  let hours = Math.floor((dist % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+  let minutes = Math.floor((dist % (1000 * 60 * 60)) / (1000 * 60));
+  let seconds = Math.floor((dist % (1000 * 60)) / 1000);
+
+  document.querySelector("#launch").innerHTML =
+    days + " d " + hours + " h " + minutes + " m " + seconds + " s ";
+
+  if (dist < 0) {
+    clearInterval(x);
+    document.querySelector("#launch").innerHTML = "EXPIRED";
+  }
+}, 1000);
+```
+
+### Conclusion
+
+---
+
+Because JavaScript engines only have one thread, asynchronous events are forced to wait in a queue for execution.
+The way setTimeout and setInterval run asynchronous code is fundamentally different.
+All of this information is highly essential and paramount.Knowing how a JavaScript engine works is a wonderful basis for constructing complex application code, especially given the enormous amount of asynchronous events that normally occur.
+
+Building a user attractive coming soon landing page will boost the rate of attracting users to register in your site always go for user sight.
+To do so, make sure your landing page is responsive, include a countdown, and include your social media links. Also, make sure your website is capable of notifying people so that they may subscribe with their email addresses to be notified when the page is ready to be launched.
+
+#### Further reading
+
+---
+

```

Comment:
Is this part complete? Remove or add some value to it, i.e giving links to other useful resources for this article.